docs/help-center/using-famedly/chat-backup-verification.mdx

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_label: Chat Backup Verification
+title: Chat Backup Verification
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 6
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Chat Backup Verification</h2>
+    <p class="hero__subtitle">How to use QR-Verification and what to do if your device can't scan the code or no other devices are available.</p>
+  </div>
+</div>
+
+
+## QR Verification
+
+You will get redirected to the QR-Verification page automatically in the login-flow, if at least one of the devices you want to use for verification is a mobile device.  
+If you nevertheless get redirected to the number- or emoji-verification instead, please check if there is an update for one of your devices. 
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Accept the verification request by clicking on <b>Continue</b> in the <b>New verification request</b> dialog.</li>
+    <li>If the other device is a mobile device, the screen on your web tab will display a QR-Code. If it's another web device, it will fallback to number verification.</li>
+    <li>Scan the QR-Code displayed on your web browser with your mobile device. If both devices are in web browsers, compare the numbers and confirm them.</li>
+    <li>If for some reason you are not able to scan the QR-Code, you can tap on <b>Compare numbers instead of QR-Code</b>.</li>
+    <li>After scanning the QR-Code, you will be asked if it was really you who scanned the QR-Code on your already active device. Click <b>Yes, it was me</b> to complete the verification.</li>
+  </ol>
+  </TabItem>
+  <TabItem value="mobile" label="Mobile">
+  <ol>
+    <li>Accept the verification request by tapping on <b>Continue</b> in the <b>New verification request</b> dialog.</li>
+    <li>One of the devices will now display a QR-Code, the other device will open a QR-Code scanner.</li>
+    <li>Please scan the displayed QR-Code.</li>
+    <li>If by some reason you are not able to scan the QR-Code, you can tap on <b>Compare numbers instead of QR-Code</b>.</li>
```

Comment:
Did we already locate the sentence 'compare numbers' more centrally? I remember in the past it was at the bottom of the page and users easily overlooked it. If it is still located at the far bottom of the page, I would add 'click/tap on compare numbers... at the bottom of the page'.

Reply:
Added "below the QR-Code scanner" to make clear where the user should search for the button.
I wouldn't change the location of the button. Probably you tested that on an iPhone 11? There was a styling bug that is fixed now that showed the button below the viewport and made the user need to scroll down to be able see it.